docs/source/overview/connect-to-database.rst

```diff
@@ -32,26 +32,23 @@ You can now preview the available data in the `restaurant_reviews` database with
 .. code-block:: python
 
    cursor.query("""
-      SELECT * 
-      FROM restaurant_reviews.food_review;
-      """).df()
+     SELECT * 
+     FROM restaurant_reviews.food_review;""").df()
 
 3. Run Native Queries in the Connected Database With `USE`
 
 You can also run native queries directly in the connected database system by the :ref:`USE<sql-use>` statement.
 
 .. code-block:: python
 
-   cursor.query(
-      """
-        USE restaurant_reviews {
-                INSERT INTO food_review (name, review) 
-                VALUES (
-                  'Customer 1', 
-                  'I ordered fried rice but it is too salty.'
-                )
-        };
-      """).df()
+   cursor.query("""
```

Comment:
Just confirming, any common indentation format that we agree upon for future docs?

Reply:
Good point. Let me see if there is a code formatting tool available for sphinx.